datahub-web-react/craco.config.js

```diff
@@ -13,6 +14,15 @@ function addLessPrefixToKeys(styles) {
 }
 
 module.exports = {
+    webpack: {
+        plugins: {
+            add: [
+                new CopyWebpackPlugin({
+                    patterns: [{ from: 'src/images', to: 'platforms' }],
+                }),
+            ],
+        },
+    },
```

Comment:
can we just add an explanation for why the copy is needed? otherwise looks great!! thanks @ngamanda

Reply:
Added!